### ISSUE TITLE
[BUG] Install fails since the new GameDB changes

### ISSUE THREAD

**Issue by alucryd** [PCSX2/pcsx2#4013]
**Describe the bug**
The build system is looking for a non-existent GameIndex.dbf since the recent GameDB changes and fails to install it.

**To Reproduce**
```
build() {
  cmake -S pcsx2 -B build -G Ninja \
    -DCMAKE_BUILD_TYPE=Release \
    -DCMAKE_INSTALL_PREFIX=/usr \
    -DDOC_DIR=/usr/share/doc/pcsx2 \
    -DGAMEINDEX_DIR=/usr/share/pcsx2 \
    -DPLUGIN_DIR=/usr/lib/pcsx2 \
    -DDISABLE_ADVANCE_SIMD=ON \
    -DDISABLE_BUILD_DATE=ON \
    -DDISABLE_PCSX2_WRAPPER=ON \
    -DENABLE_TESTS=OFF \
    -DEXTRA_PLUGINS=ON \
    -DSDL2_API=ON \
    -DPACKAGE_MODE=ON \
    -DREBUILD_SHADER=ON \
    -DUSE_LTO=OFF \
    -DUSE_SYSTEM_YAML=ON \
    -DUSE_VTUNE=OFF \
    -DXDG_STD=ON \
    -DwxWidgets_CONFIG_EXECUTABLE=/usr/bin/wx-config-gtk3 \
    -Wno-dev
  ninja -C build
}

package() {
  DESTDIR="${pkgdir}" ninja -C build install
}
```

**Expected behavior**
Files should install as usual.

**System Info (please complete the following information):**
 - PCSX2 Revision: git HEAD
 - OS: Arch Linux

**Logs and Dumps**
https://buildbot.alucryd.xyz/#/builders/327/builds/88/steps/7/logs/stdio

**Comment by tellowkrinkle** [comment]
Should've been fixed by #4008, can you confirm?

**Comment by alucryd** [comment]
I do! Thanks and sorry for the noise, the latest commits hadn't triggered a build yet, should have checked them first.
